### ISSUE TITLE
Do you plan to add fc layer?

### ISSUE THREAD

**Issue by XiaodanLi001** [leonidk/pytorch-tf#1]

**Comment by leonidk** [comment]
I don't plan to change this at all. I just wrote this when I was a TA for CS231N and had it convert squeezenet once and abandoned it. 

I just put this up in case people want to see some sketch of how this can be done.

**Comment by XiaodanLi001** [comment]
Thank you for your job. I have realized batch norm by myself and replace the fc layer with conv layer(kernel size = 1x1) and this repo works fine. Again, thank you.
